### DISCUSSION TITLE
不用自己寫程式就能開發程式

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#283] [General]
老師之前在上課有介紹google發行的teachable machine，自己可以不用寫程式，就可以開發出原本需要大量程式訓練出的模型。Microsoft(微軟)其實也有類似的工具，叫做PowerApps，可以單純用拖拉的方式寫出一個APP。
# Google的teachable machine
![image](https://user-images.githubusercontent.com/86505454/158050229-ab9a6fab-e482-4519-b76d-7b4df0bc757c.png)
# Microsoft的PowerApps
![image](https://user-images.githubusercontent.com/86505454/158050251-453ca453-5e15-4bc4-ae9f-0b4dde719428.png)

**Comment by chia-kai-chang** [comment]
power apps比較像是用網頁(免寫程式碼)開發「辦公室流程自動化」的手機APP。

power家族還有許多配套的服務可以全部串在一起。同學有興趣可以玩玩看。不過中央大學的學生office365帳號似乎不能玩power apps，你們可以嘗試一下。
